students/hyeonjin/users/views.py

```diff
@@ -34,9 +34,29 @@ def post(self, request):
                 name         = name,
                 email        = email,
                 password     = password,
-                phone_number = phone_number,
+                phone_number = phone_number
             )
             return JsonResponse({'message':'SUCCESS'}, status=201)
 
         except KeyError:
            return JsonResponse({'message': 'KEY_ERROR'}, status=400)
+
+class LoginView(View):
+
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            email    = data['email']
+            password = data['password']
+
+            usr = User.objects.get(email=email)
+            if usr.password == password:
+                return JsonResponse({"message":"SUCCESS"}, status=200)
+            else:
+                return JsonResponse({"message":"WRONG_PASSWORD"}, status=401)
```

Comment:
```suggestion
                return JsonResponse({"message":"INVALID_USER"}, status=401)
```
이메일이 틀렸을때 비밀번호가 틀렸을때 구분하지 않고 둘다 같은 메세지를 보내는게 좋을것 같습니다
다르게 처리하면 보안상 문제가 있을 것 같아요!

Reply:
오 보안 측면은 생각도 안하고 편의만 고려했어요 감사합니다!